app/serializers/v2/moves_serializer.rb

```diff
@@ -23,14 +24,9 @@ class MovesSerializer
     set_type :moves
 
     INCLUDED_FIELDS = {
-      moves: attributes_to_serialize.keys +
-        %i[profile from_location to_location prison_transfer_reason supplier],
-      profiles: ::V2::ProfileSerializer.attributes_to_serialize.keys + %i[person person_escort_record],
+      # TODO: remove these and replace with conditional relationships in relevant serializer
       people: ::V2::PersonSerializer.attributes_to_serialize.keys + %i[gender ethnicity],
       locations: ::LocationSerializer.attributes_to_serialize.keys,
-      prison_transfer_reasons: ::PrisonTransferReasonSerializer.attributes_to_serialize.keys,
-      suppliers: ::SupplierSerializer.attributes_to_serialize.keys,
-      allocations: ::AllocationSerializer.attributes_to_serialize.keys,
```

Comment:
It turns out omitting resources completely from this list means that the serializer renders everything (all attributes and relationships). Since we now use conditional relationships and custom lightweight serializers there's no need to specify which fields to render.